cli/pkg/kctrl/cmd/kctrl.go

```diff
@@ -143,6 +144,20 @@ func ConfigureGlobalFlags(o *KctrlOptions, cmd *cobra.Command, flagsFactory cmdc
 	}
 }
 
+func ConfigureTTY(o *KctrlOptions, cmd *cobra.Command) {
```

Comment:
do we need to have this public?

Reply:
Hmm, right, we can actually make the other function `ConfigureGlobal` private as well.

Reply:
Done